md/datetimes-management-tutorial.md

```diff
@@ -0,0 +1,163 @@
+# How to work with dates and times in forms and BDM
+
+::: warning
+This tutorial is for versions 7.5.0 and newer.  
+:::
+
+For use cases involving dates and times Bonita BPM comes with 3 types of data which you can manipulate through the processes, APIs and forms/pages.  
+This page gives a step by step procedure for creating a process with forms allowing to choose a date (with or without a time) and display it in another step leveraging each of these data types.  
+
+## Date only
+
+The type **DATE ONLY** is for manipulating a date which need no time information. E.g., a birth date.  
+The storage format for this type is an ISO 8601 formatted Character String (e.g. "2017-01-13").  
```

Comment:
Ok for the first and third sentence, but for the second, I would rather write
"It is stored in the database **as an ISO-8601 formated Character String** (e.g. "2017-01-13")"
Or
"using **the** using ISO-8601 "yyyy-mm-dd" **format**"